app/controllers/api/v1/accounts/statuses_controller.rb

```diff
@@ -9,7 +9,8 @@ class Api::V1::Accounts::StatusesController < Api::BaseController
   def index
     cache_if_unauthenticated!
     @statuses = load_statuses
-    render json: @statuses, each_serializer: REST::StatusSerializer, relationships: StatusRelationshipsPresenter.new(@statuses, current_user&.account_id)
+    return_source = params[:client_type] == 'editor'
+    render json: @statuses, each_serializer: REST::StatusSerializer, relationships: StatusRelationshipsPresenter.new(@statuses, current_user&.account_id), source_requested: return_source
```

Comment:
Looks like in this PR the serializer doesn't actually do anything with this value?

Reply:
Yes, that is my memory as well. It seemed like a lucky break that the code was right there to return it to clients who have valid reasons for working with it! And the idea with this particular PR was to leave open the idea that other high-level behaviors might differ for "editor" clients. I'd be happy with anything that exposes access to the raw HTML though (short of deleting it :) )